phpunit.xml.dist

```diff
@@ -17,8 +21,14 @@
         <whitelist>
             <directory>.</directory>
             <exclude>
-                <directory>Resources</directory>
+                <directory>./Resources</directory>
+                <directory>./Tests</directory>
```

Comment:
it means that you get no code coverage reports for the Tests classes. if you want those in the report, i can remove this line.